### ISSUE TITLE
Images in Vue template don't transform

### ISSUE THREAD

**Issue by koddr** [lc60005457/parcel-plugin-vue#5]
Hello.

I try to add to your [default example](https://github.com/lc60005457/parcel-plugin-vue/tree/develop/examples) image (putted on one dir with `index.vue` file), like this:

```vue
// index.vue

<template>
   <div>
      <img src="./logo.png" alt="logo">
      Hello {{ World }}!<ComponentA />
   </div>
</template>
...
```

But if I run `parcel index.html` — image don't placed in `dist` dir and address stay `./logo.png`. 

If I put image to `index.html` file — all okay. How can fix it? Please.

**Comment by BoltDoggy** [comment]
Waiting for https://github.com/parcel-bundler/parcel/issues/326

**Comment by koddr** [comment]
@lc60005457 thanks for fast reaction! Yep, just wait for @devongovett

**Comment by masonz** [comment]
I've had the same problem before, my solution is to use `import './logo.png'`.
### Code Snippets:
```
// index.vue

<template>
   <div>
      <img :src="logo" alt="logo">
   </div>
</template>
<script>
import Logo from './logo.png';

export default {
    data() {
        return {
            logo: Logo
        }
    }
};
</script>
...
```

**Comment by koddr** [comment]
@masonz it's okay if you have one-two-...-ten image in Vue component, I use this. But it goes to nightmare if you have huge component with many libraries imports and images and no way to separate it.

**Comment by BoltDoggy** [comment]
@koddr @masonz I published the 'parcel-plugin-vue@1.4.0'.

`<img src="...">` will be replaced to `<img src="require(...)">` always.

---

But you must use local `parcel-bundler@latest` not global.

And you should use '--public-url /', so you must visit the full url 'http://127.0.0.1:1234/index.html', not 'http://127.0.0.1:1234/'.

The reason is not certain,  I will fix it as soon as possible, or response issues for 'parcel-bundler'.

**Comment by masonz** [comment]
@lc60005457 I specified the **output directory** and the **public URL** at the same time. (e.g.`parcel src/index.html -d dist --public-url /dist/`), now that's normal. But I still need to use local `parcel-bundler`.
